src/machine.rs

```diff
@@ -230,27 +230,27 @@ impl<'m> Machine<'m> {
         if let Err(e) = self
             .setup_vm()
             .and_then(|_| async {
-                trace!("{vm_id}: Booting the VM instance...");
+                debug!("{vm_id}: Booting the VM instance...");
 
                 self.send_action(Action::InstanceStart).await
             })
             .await
         {
-            trace!(
+            debug!(
                 "{vm_id}: Failed to boot VM instance: {}. Force shutting down..",
                 e
             );
             self.force_shutdown().await.unwrap_or_else(|e| {
                 // `force_shutdown` only updates the state on success.
                 self.state = MachineState::SHUTOFF;
                 // We want to return to original error so only log the error from shutdown.
-                trace!("{vm_id}: Failed to force shutdown: {}", e);
+                debug!("{vm_id}: Failed to force shutdown: {}", e);
             });
 
             return Err(e);
         }
 
-        trace!("{vm_id}: VM started successfully.");
+        debug!("{vm_id}: VM started successfully.");
```

Comment:
:-1:  here too. The caller knows about the success so they can log the success on any level they prefer.